packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -509,8 +498,7 @@ class AccountSwitcher extends React.Component {
                                                 secondary
                                                 small
                                                 is_disabled={
-                                                    ((!this.props.is_eu_enabled || !this.props.is_eu) && // TODO [deriv-eu] remove is_eu_enabled check once EU is ready for production
-                                                        !this.props.has_any_real_account) ||
+                                                    !this.props.has_any_real_account ||
```

Comment:
Not sure about this, but, I think we should keep the `!this.props.is_eu` here, since if the client has `MLT` and wants to create a `MF` one, it will be disabled?